Cargo.lock

```diff
@@ -1735,6 +1735,15 @@ dependencies = [
  "libc",
 ]
```

Comment:
That's fine, its not a dependency of wasmtime, and we can snip that dependency in a follow up PR if desired.

Reply:
Looks like the region PR for moving to mach2 has been open for several months: https://github.com/darfink/region-rs/pull/27 And worse the last release has been from over 2 years ago and once the current version gets released, it will lock the libc crate to version 0.2.107 from 2 years ago (for OpenBSD 6.9 support, which has been EOL since May 2022. libc presumably dropped support for it because it is EOL)